src/pages/index.jsx

```diff
@@ -1,30 +1,10 @@
 import React from "react";
-import { graphql, useStaticQuery, Link as GatsbyLink } from "gatsby";
+import { graphql, useStaticQuery} from "gatsby";
 import Layout from "src/layout";
-import { makeStyles } from "@material-ui/core/styles";
-import Typography from "@material-ui/core/Typography";
-import Box from "@material-ui/core/Box";
-import Card from '@material-ui/core/Card';
-import CardActionArea from '@material-ui/core/CardActionArea';
-import CardContent from '@material-ui/core/CardContent';
+import ArticleList from "../components/article-list";
 
-const useStyles = makeStyles((theme) => ({
-  link: {
-    textDecoration: "none",
-  },
-  divider: {
-    background: theme.palette.text.secondary,
-  },
-  article: {
-    "& ~ *": {
-      "border-top": `1px solid ${theme.palette.text.secondary}`,
-    },
-  },
-}));
 
 const Index = () => {
-  const classes = useStyles();
-
   const recent = useStaticQuery(graphql`
```

Comment:
Is there any way to limit this to a certain number of recent articles so it doesn't end up being too long as more publications are added?

Reply:
I didn't read the tutorial on pagination but presumably yes it's possible to have a shorter list. I'll open an issue for it.

Reply:
It's possibly, but let's leave that for when we implement pagination.